tests/rbd/test_rbd_groups_image_clone.py

```diff
@@ -194,19 +216,48 @@ def test_rbd_groups_image_clone(rbd_obj, client, **kw):
 
                 # 12. Map the cloned images as a new block disk
                 log.info(f"Map the cloned image {clone} and run IO")
-                image_spec = f"{pool}/{clone}"
-                size = kw.get("size")
-                file_path = f"/tmp/{clone}_dir/{clone}_file"
-                mount_config = {
-                    "rbd": rbd,
-                    "client": client,
-                    "size": size,
-                    "file_path": file_path,
-                    "image_spec": image_spec,
-                    "io": True,
-                }
-                if map_and_mount_image(**mount_config):
-                    log.error(f"Map and mount failed for {image_spec}")
+                io_config["config"]["image_spec"] = [f"{pool}/{clone}"]
+                krbd_io_handler(**io_config)
+
+                # Remove the group snapshot snap_name from the group.
+                log.info(f"Remove group snapshot {group_snap}")
+                rbd.group.snap.rm(pool=pool, snap=group_snap)
+                # Verify that the snapshot has been moved to the trash.
+                log.info(f"Verify that the group snapshot {group_snap} is removed")
+                out, err = rbd.snap.ls(pool=pool, image=image, all="", format="json")
+                snap_json = json.loads(out)
+                for snap in snap_json:
+                    if snap["name"] == group_snap:
+                        log.error(
+                            f"Group snapshot {group_snap} found even after deletion"
+                        )
+                        return 1
+                # Verify that the cloned image still exists.
+                log.info(f"Validate clone image {clone} still exist")
+                info_spec = {"image-or-snap-spec": f"{pool}/{clone}", "format": "json"}
+                out, err = rbd1.info(**info_spec)
+                if err:
+                    log.error(f"Error while fetching info for image {clone}")
+                    return 1
+                out_json = json.loads(out)
+                if out_json["name"] != clone:
+                    log.error(f"clone image {clone} does not exist")
+                    return 1
+
+                # Unmap the cloned image.
+                log.info(f"Unmap clone image: {clone}")
+                rbd.unmap(pool=pool, image=clone)
+
+                # Attempt to delete the cloned image.
+                log.info(f"Delete clone image: {clone}")
+                rbd.rm(pool=pool, image=clone)
+                # Verify that the cloned image has been removed and is no longer listed. E.g: rbd ls testpool
+                log.info(f"Verify cloned image {clone} has been removed")
+                out, err = rbd.ls(pool=pool, format="json")
+                out_json = json.loads(out)
+                log.info(out_json)
+                if clone in out_json:
+                    log.error(f"clone image {clone} still exist even after deleting")
```

Comment:
please add `log.info` for the successfull deletion.

Reply:
Done